### ISSUE TITLE
Test all Supernodes with testsnXX nodes 

### ISSUE THREAD

**Issue by lemoer** [freifunkh/ansible#166]
- [ ] Automate the creation of testsnXX nodes
- [ ] Deploy testsnXX for fastd for every supernode
- [ ] Deploy testsnXX for wireguard for every supernode

I started to implement the creation of testsnXX nodes here: https://github.com/freifunkh/ansible-deploy-gluon-to-proxmox .

**Comment by lemoer** [comment]
I added the milestone "Beginn der stabilen Phase", as this contains useful monitoring and should therefore implemented before the "stabile Phase".

**Comment by AiyionPrime** [comment]
This contains monitoring? :)

**Comment by 1977er** [comment]
Yes. We already had some testsnXX nodes, which were set up manually and probably did not survive an autoupdate.

The former testsn nodes included a zabbix-client on the gluon node.

**Comment by lemoer** [comment]
I made some progress today:
- A proxmox bridge for the client interface of the router is created.
- The bridge is used to configure the node via ssh.
- Currently the following things are done in config mode:
    - The hostname are configured correctly.
    - The node is marked as "configured".
    - Afterwards the node is rebooted.

**Comment by lemoer** [comment]
I again made some progress today:
- Multiple gluon nodes can now be created.
- The fastd/wireguard keys are downloaded to the ansible control host, so they can be added to a peers git later.

@1977er After the current playbook is executed, the nodes are rebooted into normal running mode. Therefore if their keys are submitted into our peers directory, they should already be fully functional. Other tasks/playbooks could then install zabix, ... stuff. 

Next steps:
- [ ] Maybe use [add_host](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/add_host_module.html) and [delegate_to](https://docs.ansible.com/ansible/latest/user_guide/playbooks_delegation.html) to run stuff on the newly created hosts.

**Comment by lemoer** [comment]
This will be normal gluon nodes, which are added to the peers repo. Therefore this can be done without touching any core infrastructure. Therefore it's not a blocker for the milestone.

**Comment by lemoer** [comment]
Seems like somebody has archived https://github.com/freifunkh/ansible-deploy-gluon-to-proxmox today. Let's close this.
